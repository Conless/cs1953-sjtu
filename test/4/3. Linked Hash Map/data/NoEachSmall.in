0
10 0 10 
0 3 msYpXaucUxnSkr3oL2ei 
1 3 msYpXaucUxnSkr3oL2ei 
0 3 6Oz3KnAyWh7oCIwiQg9X 
2 3 
1 3 6Oz3KnAyWh7oCIwiQg9X 
0 4 S3JOPvkb179IXluq8m6c 
1 4 S3JOPvkb179IXluq8m6c 
0 2 6wavDL8oE2X07cSRkOPp 
1 5 DmQCHW7gFvKNSUMT2VEj 
2 2 
