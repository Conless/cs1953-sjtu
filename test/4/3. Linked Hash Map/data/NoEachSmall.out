6Oz3KnAyWh7oCIwiQg9X 
6wavDL8oE2X07cSRkOPp 
